src/imitation/scripts/config/train_adversarial.py

```diff
@@ -263,7 +247,7 @@ def fast():
 # Shared settings
 
 ant_shared_locals = dict(
-    n_epochs=2000,
+    timesteps=3e7,
```

Comment:
This is a lot of timesteps! but maybe we need it. weird Humanoid gets less timesteps than this.